parser/utils.go

```diff
@@ -49,3 +49,10 @@ func printHelp(argv []string, usage string) bool {
 
 	return false
 }
+
+func addGlobalFlags(input string) string {
```

Comment:
An alternative approach would be to parse it from the global section and pass it as an argument. In that case however, it would not show up anywhere except `deis help`, so I though it wouldn't have enough visibility.